outis-core/src/main/scala/io/saagie/outis/core/anonymize/Anonymize.scala

```diff
@@ -0,0 +1,34 @@
+package io.saagie.outis.core.anonymize
+
+import scala.util.Random
+
+object Anonymize {
+
+  val letters = ('a' to 'z') ++ ('A' to 'Z')
+  val random = Random
+
+  def suppression(): String = {
```

Comment:
Spark will take care of the null type, this function should return an option with None value